openff/recharge/cli/reconstruct.py

```diff
@@ -144,4 +113,26 @@ def reconstruct(
         },
     )
 
-    esp_store.store(*esp_records)
+    with ProcessPoolExecutor(
+        max_workers=n_processors, mp_context=get_context("spawn")
+    ) as pool:
+
+        futures = [
+            pool.submit(
+<<<<<<< HEAD
+                _process_result(qc_result, grid_settings=grid_settings),
+            )
+            for qc_result in qc_results
+        ]
+        #to avoid simultaneous writing to the db, wait for each calculation to finish then write
+=======
+                functools.partial(_process_result, grid_settings=grid_settings),
+                qc_result,
+            )
+            for qc_result in qc_results
+        ]
+
+>>>>>>> 08ccc816eecf17529efc6e53bb2630c3d8885668
```

Comment:
Accidental merge conflict committed here (no worries - I assure you I've done this more times you ever will!)

Reply:
Oops my bad!